src/include/rados.h

```diff
@@ -116,6 +116,7 @@ struct ceph_eversion {
 #define CEPH_OSD_NEW     (1<<3)  /* osd is new, never marked in */
 #define CEPH_OSD_FULL    (1<<4)  /* osd is at or above full threshold */
 #define CEPH_OSD_NEARFULL (1<<5) /* osd is at or above nearfull threshold */
+#define CEPH_OSD_BACKFILL_FULL (1<<6) /* osd is at or above backfill_full threshold */
```

Comment:
This is used for to get the health warning and to notify the mon about the local OSDs state.